src/app/[locale]/specs/permission/en.mdx

```diff
@@ -0,0 +1,347 @@
+
+Resources and permissions are described in the abstract, not tied to specific PDS API endpoints. For example, `rpc` permissions describe the ability to make remote service API calls using inter-service authentication. This includes both proxied PDS requests (using HTTP headers) and fetching service auth tokens using `com.atproto.server.getServiceAuth`.
+
+### `repo`
+
+Write access to records in the account's public repository. Can be limited to specific record types (collections) or actions (eg update vs delete).
+
+Parameters:
+
+- `collection` (string array, required, positional): NSID of record types. Wildcard (`*`) is allowed and gives access to all records. Partial wildcards are not supported (eg, can not use `com.example.*`)
+- `action` (string array, optional): defines the set of record operations allowed. Allowed values are `create`, `update`, `delete`. If not defined, all operations are allowed
+
+Examples:
+
+```
+# full permission (create, update, delete) on the Bluesky profile record type
+repo:app.example.profile
+
+{
+  "type": "permission",
+  "resource": "repo",
+  "collection": ["app.example.profile"]
+}
+
+# same as above, with actions being explicit
+repo:app.example.profile?action=create&action=update&action=delete
+
+{
+  "type": "permission",
+  "resource": "repo",
+  "collection": ["app.example.profile"],
+  "action": ["create", "update", "delete"]
+}
+
+# full permissions on all record types
+repo:*
+
+{
+  "type": "permission",
+  "resource": "repo",
+  "collection": ["*"]
+}
+
+# delete only permission on all record types
+repo:*?action=delete
+```
+
+### `rpc`
+
+The ability to make authenticated API requests to remote services. This includes both requesting service auth tokens (JWTs) using the `com.atproto.server.getServiceAuth` endpoint on the PDS, and requests proxied via the PDS.
+
+The permission is parameterized by the remote endpoint (`lxm`, short for "Lexicon Method") and the identity of the remote service (the audience, `aud`). Permissions must be restricted by one or the other: the ability to call any API endpoint on any service can not be declared with a single permission.
+
+When included in a permission set, the audience parameter may be inherited from the overall set.
+
+Parameters:
+
+- `lxm` (string array, required, positional): NSID of API endpoints. Wildcard (`*`) is allowed and gives access to all endpoints. Partial wildcards are not supported (eg, can not use `com.example.*`).
+- `aud` (string, semi-required): audience of API requests, as a DID service reference: DID followed by required service type fragment, eg `did:web:api.example.com#srvtype`). Supports wildcard (`*`). Required in granular string representation, and required in lexicon JSON unless `inheritAud` is true.
+- `inheritAud` (boolean, optional): if true, `aud` does not need to be defined.
```

Comment:
maybe move the note from about down to here & note that this is only relevant in the context of a permission set